dashboard/client/src/pages/overview/cards/RecentJobsCard.component.test.tsx

```diff
@@ -60,4 +62,21 @@ describe("RecentJobsCard", () => {
     expect(screen.getByText("06000000")).toBeVisible();
     expect(screen.queryByText("07000000")).toBeNull();
   });
+  it("the link is active when job_id is not null", async () => {
+    render(<RecentJobsCard />, { wrapper: MemoryRouter });
+
+    await screen.findByText("01000000");
+
+    const link = screen.getByRole("link", { name: "05000000" });
+    expect(link).toHaveAttribute("href");
+  });
+  it("disables link when job_id is null", async () => {
+    render(<RecentJobsCard />, { wrapper: MemoryRouter });
+
+    await screen.findByText("01000000");
+    const jobDiv = screen.getByText("raysubmit_23456");
+
+    fireEvent.click(jobDiv);
+    expect(window.location.pathname).not.toBe("/jobs/null");
```

Comment:
instead of this, can we do `expect(screen.getByRole("link", {name: "raysubmit_23456"})).not.toBeVisible()`

Reply:
Good idea, queryByRole is a more normal way for test case